activation/activation.go

```diff
@@ -272,8 +259,12 @@ func (b *Builder) StopSmeshing(deleteFiles bool) error {
 	}
 
 	b.stop()
-	b.status = smeshingStatusIdle
-
+	select {
+	case <-b.exited:
+		b.exited = nil
+	case <-ctx.Done():
```

Comment:
this is a bit weird that we had to check for ctx.Done() here, since this context can be any random one?
if we don't wait for b.exited to be closed here, the next call to StartSmeshing() will not be successful.

Reply:
thanks, this is clearly a bug